brain-marks.xcodeproj/project.pbxproj

```diff
@@ -23,6 +23,7 @@
 		C4392E80BFE640DAA2625F9D /* AWSTweet+Schema.swift in Sources */ = {isa = PBXBuildFile; fileRef = CC43813A9098469FB57F5871 /* AWSTweet+Schema.swift */; };
 		CA80832580B84CB3A2D5F629 /* AmplifyModels.swift in Sources */ = {isa = PBXBuildFile; fileRef = 14F5BDE61EAA4A2DB9030734 /* AmplifyModels.swift */; };
 		CF16AFB9BE3F401E98C85114 /* AWSTweet.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0AA54EDF3B34443CA9D530E5 /* AWSTweet.swift */; };
+		D382A99228E91664007BAE5F /* Secrets.swift in Sources */ = {isa = PBXBuildFile; fileRef = D382A99128E91664007BAE5F /* Secrets.swift */; };
```

Comment:
You added the `Secrets.swift` file in some other way, so it shows up here when it shouldn't, I'd suggest taking a look at my updated instructions [here](https://github.com/mikaelacaron/brain-marks/blob/dev/CONTRIBUTING.md#setting-up-secrets)

Reply:
I cloned the base repo, added the `Secrets.swift` file via terminal, added the changes and pushed the code again. Hopefully, it should all be good now.

Reply:
Good work!! There's one string missing `"Done"` that's used for editing. I can still merge this, but if you'd be interested you can add the english / german / french translations

Reply:
Done!

I had to look up the german one since I don't speak German. I checked multiple sites to get a better translation for "Done", "Done button", "Finished" etc. This translation seems to work but I'm not 100% sure. The French one seems good to go, at least in the contexts it's been used at the moment.

Reply:
Awesome! I'll take a look again tomorrow
You'll be the first merged PR of Hacktoberfest 2022! ❤️